frontend/src/app/components/record-rtc/record-rtc.component.ts

```diff
@@ -0,0 +1,97 @@
+import { Component, OnInit, ViewChild, AfterViewInit } from '@angular/core';
+
+declare var require: any;
+
+const recordRTCDependency = require('recordrtc/RecordRTC.min'); // tslint:disable-line:max-line-length
```

Comment:
This tslint instruction is not needed here.

Reply:
Deleted. Thanks.